src/azul/service/hca_response_v5.py

```diff
@@ -343,6 +343,10 @@ def make_projects(self, entry):
                 "projectShortname": project["project_short_name"],
                 "laboratory": sorted(set(project.get("laboratory", [None])))
             }
+            if self.entity_type in ('projects', 'bundles'):
+                entity = one(entry['contents']['entity_dates'])
```

Comment:
This passes the IT? How can it be that there is only one inner entity in the aggregate if the contributions have many and there is no aggregation that combines them.

Reply:
The `entity_dates` inner entities are aggregated with a [DateAggregator](https://github.com/DataBiosphere/azul/blob/eda69d70f01091d23c744c0572301913894228d6/src/azul/plugins/metadata/hca/aggregate.py#L205) which makes use of the fact that its grand-parent the [EntityAggregator](https://github.com/DataBiosphere/azul/blob/eda69d70f01091d23c744c0572301913894228d6/src/azul/indexer/aggregate.py#L350) already has conditions to handle the fields `submission_date` and `update_date` with a `MinAccumulator` and `MaxAccumulator` respectively.